projects/driver_controls/inc/input_event.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
```

Comment:
It's still used by the digital_io and analog_io modules, so I needed to keep it in.